circle.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
 
 test:
   override:
-    - ./gradlew clean :core:install :core:verify :mapzen-android-sdk:verify :mapzen-places-api:verify
+    - ./gradlew clean :core:install :core:verify :mapzen-android-sdk:verify :mapzen-places-api:verify --configure-on-demand
```

Comment:
👍 Nicely documented in gradle docs (which sometimes are very convoluted, at least for me): https://docs.gradle.org/current/userguide/multi_project_builds.html#sec:configuration_on_demand